allennlp/tests/common/registrable_test.py

```diff
@@ -73,9 +73,10 @@ class FakeAlternate(base_class):
 
     # TODO(mattg): maybe move all of these into tests for the base class?
 
-    def test_registry_has_builtin_iterators(self):
-        assert DataIterator.by_name("basic").__name__ == "BasicIterator"
-        assert DataIterator.by_name("bucket").__name__ == "BucketIterator"
+    def test_registry_has_builtin_samplers(self):
```

Comment:
Let's just remove this.  I don't think these "has builtin" tests are useful.  They were added a very long time ago, before the registry works the way it does.